src/lib/events/offlineRsvpSync.ts

```
@@ -0,0 +1,60 @@
+import { get, set } from 'idb-keyval';
```

ESLint

[error] Replace `'idb-keyval'` with `"idb-keyval"`

(prettier/prettier)